utils/create_validate_pr_quickstarts.ts

```diff
@@ -111,6 +111,20 @@ export const createValidateQuickstarts = async (
     })
     .filter(Boolean);
 
+  const quickstartsWithInstallPlans = quickstarts
+    .map((qs) => {
+      return qs.config?.installPlans?.length ? qs.identifier : undefined;
+    })
+    .filter(Boolean);
+
+  if (quickstartsWithInstallPlans.length > 0) {
+    logger.info(
+      `WARNING: Found quickstart(s) with install plans.\n` + 
```

Comment:
Would it make sense to link to some relevant documentation/context in this warning?